libmamba/CMakeLists.txt

```diff
@@ -301,36 +301,38 @@ set(
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/iterator.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/json.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/loop_control.hpp
-    ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os_linux.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os_osx.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os_unix.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os_win.hpp
+    ${LIBMAMBA_INCLUDE_DIR}/mamba/util/os.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/parsers.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/path_manip.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/random.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/string.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/tuple_hash.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/type_traits.hpp
-    ${LIBMAMBA_INCLUDE_DIR}/mamba/util/url.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/url_manip.hpp
+    ${LIBMAMBA_INCLUDE_DIR}/mamba/util/url.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/variant_cmp.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/util/weakening_map.hpp
     # Implementation of version and matching specs
     ${LIBMAMBA_INCLUDE_DIR}/mamba/specs/archive.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/specs/authentication_info.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/specs/build_number_spec.hpp
     ${LIBMAMBA_INCLUDE_DIR}/mamba/specs/channel.hpp
+    ${LIBMAMBA_INCLUDE_DIR}/mamba/specs/chimera_string_spec.hpp
```

Comment:
This is an example of missing line. Everything will work except that VS Code will give errors when you open this file - it annoyed me, so I decided to create this PR